DeDriftAndResample/DeDriftAndResamplePipeline.sh

```diff
@@ -60,30 +60,34 @@ PARAMETERs are [ ] = optional; < > = user supplied value
-   [--concat-reg-name=<regname>] String corresponding to the output name of the dedrifted registration
+  [--concat-reg-name=<regname>] String corresponding to the output name of the dedrifted registration
    --maps=<non@myelin@maps> @-delimited map name strings corresponding to maps that are not myelin maps 
        (e.g. sulc@curvature@corrThickness@thickness)
-   [--myelin-maps=<myelin@maps>] @-delimited map name strings corresponding to myelin maps 
+  [--myelin-maps=<myelin@maps>] @-delimited map name strings corresponding to myelin maps
        (e.g. MyelinMap@SmoothedMyelinMap). No _BC, this will be reapplied.
-   [--multirun-fix-names=<day1run1@day1run2%day2run1@day2run2>] @ and % delimited list of lists of
+  [--multirun-fix-names=<day1run1@day1run2%day2run1@day2run2>] @ and % delimited list of lists of
        fMRIName strings that will have multi-run ICA+FIX reapplied to them (could be either rfMRI or
        tfMRI). Requires specifying --multirun-fix-concat-names also, with same number of concat names as
        lists of runs in this option.
-   [--multirun-fix-concat-names=<day1_concat@day2_concat>] @-delimited list of names of the concatenated
+  [--multirun-fix-concat-names=<day1_concat@day2_concat>] @-delimited list of names of the concatenated
        timeseries, only required when using --multirun-fix-names.
-   [--multirun-fix-extract-names=<day1run1@day1run2%day2run1@day2run2>] @ and % delimited list of lists of
+  [--multirun-fix-extract-names=<day1run1@day1run2%day2run1@day2run2>] @ and % delimited list of lists of
        fMRIName strings to extract, one list for each multi-run ICA+FIX group in --multirun-fix-names (use
        NONE to skip a group), only required when using --multirun-fix-extract-concat-names.  Exists to
        enable extraction of a subset of the runs in a multi-run ICA+FIX group into a new concatenated
        series (which is named using --multirun-fix-extract-concat-names).
-   [--multirun-fix-extract-concat-names=<day1_newconcat@day2_newconcat>] @-delimited list of names for the
+  [--multirun-fix-extract-concat-names=<day1_newconcat@day2_newconcat>] @-delimited list of names for the
        concatenated extracted timeseries, one for each multi-run ICA+FIX group (i.e. name in
        --multirun-fix-concat-names; use NONE to skip a group).
-   [--fix-names=<ICA+FIXed@fMRI@Names>] @-delimited fMRIName strings corresponding to maps that will 
+  [--multirun-fix-extract-extra-regnames=<regname@regname>] extract MR FIX runs for additional surface
+       registrations, often MSMSulc.
+  [--multirun-fix-extract-volume={TRUE, FALSE}] whether to also extract the specified MR FIX runs from the
+       volume data, requires --multirun-fix-extract-concat-names to work, default FALSE.
+  [--fix-names=<ICA+FIXed@fMRI@Names>] @-delimited fMRIName strings corresponding to maps that will
        have ICA+FIX reapplied to them (could be either rfMRI or tfMRI). Previously known as --rfmri-names.
```

Comment:
```suggestion
  [--fix-names=<ICA+FIXed@fMRI@Names>] @-delimited fMRIName strings corresponding to maps that will
       have ICA+FIX reapplied to them (could be either rfMRI or tfMRI). Previously known as --rfmri-names.
       Not relevant if using multirun-fix.
```

Reply:
I think it is technically possible to have some runs in a subject that are handled with MR FIX, and others that aren't - it isn't what we recommend, though.

Reply:
Well then, instead of my initial suggestion, how about "Relevant for runs processed with single-run FIX" instead?

Reply:
How does this look:
```
  [--fix-names=<ICA+FIXed@fMRI@Names>] @-delimited fMRIName strings corresponding to maps that will
       have single-run ICA+FIX reapplied to them (could be either rfMRI or tfMRI). Do not specify runs
       processed with MR FIX here. Previously known as --rfmri-names.
  [--dont-fix-names=<not@ICA+FIXed@fMRI@Names>] @-delimited fMRIName strings corresponding to maps that
       will not have ICA+FIX reapplied to them (not recommended, MR FIX or at least ICA+FIX is recommended
       for all fMRI data). Previously known as --tfmri-names.
```

Reply:
So, in the parenthetical for `--dont-fix-names`, it seems that you are equating "ICA+FIX" with _single-run_ ICA+FIX?  I think there is the potential for some confusion yet in how `--dont-fix-names` relates to the single vs. multi-run FIX issue.

I don't suppose that we have any error checking built in that a run isn't listed in both `--multirun-fix-names` and `--dont-fix-names`?

Reply:
We don't explicitly check for a name in more than one list, no.  I can add "single-run" to the description there.